packages/cli/src/commands/doctor/versionRanges.ts

```diff
@@ -4,6 +4,7 @@ export default {
   YARN: '>= 1.10.x',
   NPM: '>= 4.x',
   WATCHMAN: '4.x',
+  PYTHON: '>= 2.x <= 3',
```

Comment:
I believe we want to make sure people are on Python2, right? I'm not sure if Python3 is supported.
```suggestion
  PYTHON: '>= 2.x < 3',
```